service/history/shard/context.go

```diff
@@ -180,11 +180,12 @@ var (
 )
 
 const (
-	conditionalRetryCount    = 5
-	logWarnTransferLevelDiff = 3000000 // 3 million
-	logWarnTimerLevelDiff    = time.Duration(30 * time.Minute)
-	historySizeLogThreshold  = 10 * 1024 * 1024
-	minContextTimeout        = 1 * time.Second
+	conditionalRetryCount       = 5
+	logWarnTransferLevelDiff    = 3000000 // 3 million
+	logWarnTimerLevelDiff       = time.Duration(30 * time.Minute)
+	logWarnCrossClusterLevelLag = 3000000
```

Comment:
is this a duration? if so, do you mind commenting the unit of this?

Reply:
It's not duration, but an absolute number based on for taskID which is calculated based on rangeID. So it's a very rough estimate, during shard movement rangeID will increase by 1 and taskID will be increased by around 1M. I basically used the same number as `logWarnTransferLevelDiff`.

Reply:
Can we use a more descriptive name like minTaskIDToLogWarn or add a comment here to explain what this is?

Reply:
Added a comment for transfer and cross-cluster level